components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -435,6 +444,9 @@ def rename(self, args):
 template. By default the original files and import log are also
 moved.
 """
+        if platform.system() == 'Windows':
```

Comment:
I was using the same condition as on of the other examples I looked at but happy to chnage if there is a preferred style. /cc @jburel